cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -193,7 +194,7 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, PlatformFeature
             }
 
             Promise<Void> returnPromise = Promise.promise();
-            CompositeFuture.all(futures).onComplete(res -> {
+            CompositeFuture.all(new ArrayList<>(futures)).onComplete(res -> {
```

Comment:
Same as above ... Why do we need to create a new list here?

Reply:
Because `CompositeFuture.all` required type` List<Future>`